modules/auxiliary/gather/nis_ypserv_map.rb

```diff
@@ -140,7 +136,11 @@ def parse_map(res)
           String   # Key: [redacted]
         )
 
-        status == 1 ? map[key] = value : break
+        if status == 1 && key && value
+          map[key] = value
+        else
+          break
+        end
```

Comment:
I'm not sure your motivation for this style change. Presumably to prevent using `break` in a one line conditional.

This might be cleaner :

```ruby
        break unless status == 1 && key && value

        map[key] = value
```